hugegraph-core/src/main/java/com/baidu/hugegraph/task/ServerInfoManager.java

```diff
@@ -337,7 +335,7 @@ private HugeServerInfo serverInfo(Id server) {
     }
 
     private HugeServerInfo removeSelfServerInfo() {
-        if (this.initialized()) {
+        if (this.graph.initialized()) {
```

Comment:
should use this.call() if graph.initlalized() will open tx (ensure graph tx closed when graph close)